### PR TITLE
Use new postgres config scheme

### PR DESCRIPTION
**Pull request by carbonin**
This is a complementary PR to https://github.com/ManageIQ/manageiq-appliance/pull/116

This uses the newly named `postgresql.conf` file and makes dynamic changes to the configuration using `ALTER SYSTEM` SQL commands rather than relying on reference files and ERB.

### CODE REVIEW

File: lib/gems/pending/appliance_console/internal_database_configuration.rb [hunk 1]
```diff
@@ -126,25 +130,49 @@ def start_postgres
...
+      yield conn
+    ensure
+      conn.close
+    end
+
+    def apply_initial_configuration
+      shared_buffers = run_as_evm_server ? SHARED_DB_SHARED_BUFFERS : DEDICATED_DB_SHARED_BUFFERS
+      with_pg_connection do |conn|
+        conn.exec("ALTER SYSTEM SET ssl TO on") if ssl
+        conn.exec("ALTER SYSTEM SET shared_buffers TO #{shared_buffers}")
```
Comment (gtanzillo) [inline lib/gems/pending/appliance_console/internal_database_configuration.rb]: If we're setting shared_buffers this this way, with `ALTER SYSTEM`, is it still necessary to have it in `01_miq_overrides.conf`?

Also, are these the only ones we want to set this way or are you planning to add more later?

Comment (carbonin) [inline lib/gems/pending/appliance_console/internal_database_configuration.rb]: As of now this is just a replacement for the old reference conf files (`EVM_V5_Dedicated_Appliance_postgresql.conf`/`EVM_V5_Shared_Appliance_postgresql.conf`).

The only difference between these and the `postgresql.conf` that was used by default was the shared_buffers setting. So now that we know if we are a dedicated database appliance or not we can just apply this directly.

Comment (carbonin) [inline lib/gems/pending/appliance_console/internal_database_configuration.rb]: Also, yeah, we could remove it from the `conf` file.

Comment (carbonin) [inline lib/gems/pending/appliance_console/internal_database_configuration.rb]: Done in https://github.com/ManageIQ/manageiq-appliance/pull/116/commits/77764f0b043205b4839732e2a3b0a05f137f3e56

**Comment by carbonin** [comment]
Successfully tested this out on an appliance in the shared and dedicated cases.

Removing WIP.

**Pull request merged**